### DISCUSSION TITLE
Wrong Docs for Remote Schema Response Caching

### DISCUSSION THREAD

**Discussion by vangelov** [hasura/graphql-engine#8216] [General] [answered]
I noticed a strange contradiction between the website the docs for Hasura Cloud:

On the website (https://hasura.io/graphql/caching/) it says "Hasura Cloud supports query response caching for Remote Schemas"

However, according to the docs (https://hasura.io/docs/latest/graphql/cloud/response-caching.html) the following is true:

"A query’s response can be cached only if the following conditions hold:

The query does not make use of remote schemas or remote joins"

Does anyone know if caching is actually supported for remote schema responses? Thanks!

**Comment by sordina** [comment, marked as answer]
Hi @vangelov 

The relevant validation performed is:

* ttlDirective > maxTTL:
    `NotSupported: "The maximum allowed TTL is {{maxTTL}} seconds"`
* hasRemoteJoinsForwardingHeaders:
    `NotSupported: "Remote joins that forward client headers cannot currently be cached"`
* hasActionsFowardingHeaders:
    `NotSupported: "Actions that forward client headers cannot currently be cached"`

With

`hasRemoteJoinsForwardingHeaders`  looking for any `vrsdFwdClientHeaders` in the associated remoteSchemas
and `hasActionsFowardingHeaders` looking for any `_asiForwardClientHeaders` in the associated actions

So I believe that this should allow caching actions and remote schemas without forwarded headers in Hasura cloud.

Therefore I believe that the documentation will need to be updated. We will confirm this and update the docs if it is indeed incorrectly documented.

**Reply by vangelov** [reply]
Thanks for your answer. I also asked the same question in Discord and got this reply from one of your team members (praveenweb):

> Hey! What you see on the website is accurate. Query response caching is available for remote schemas (and Actions). The docs, unfortunately, seem to be out of date. We will fix that now 🙂

**Reply by sordina** [reply]
@paritosh-08 did this end up being resolved?
